automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/summary/WorkspaceSummaryPage.scala

```diff
@@ -17,6 +17,8 @@ import org.scalatest.selenium.Page
 class WorkspaceSummaryPage(namespace: String, name: String)(implicit webDriver: WebDriver)
   extends WorkspacePage(namespace, name) with Page with PageUtil[WorkspaceSummaryPage] with Stateful {
 
+  wsSummaryPage: WorkspaceSummaryPage =>
```

Comment:
Unnecessary

Reply:
to remove it, how do I then access outer `awaitReady` from inside of sidebar?